workitem/workitemtype_repository.go

```diff
@@ -26,6 +26,7 @@ type WorkItemTypeRepository interface {
 	CreateFromModel(ctx context.Context, model *WorkItemType) (*WorkItemType, error)
 	List(ctx context.Context, spaceID uuid.UUID, start *int, length *int) ([]WorkItemType, error)
 	ListPlannerItems(ctx context.Context, spaceID uuid.UUID) ([]WorkItemType, error)
+	LoadByID(ctx context.Context, id uuid.UUID) (*WorkItemType, error)
```

Comment:
Thanks, correct.
I added that earlier when we have `included` section for WIT..
removed now in https://github.com/fabric8-services/fabric8-wit/pull/1541/commits/b4c92e120d865951aded9eb8e2d3cbf68525d283